### PR TITLE
Fix minor typos and installation in README.md

### PR DESCRIPTION
**Pull request by Benjamin-eecs**
"python setup.py install" actually has installation errors
either "python setup.py develop" or "pip install -e ."

### CODE REVIEW

**Comment by vmoens** [comment]
Thanks for this! Could you report the error? Perhaps I can fix it

**Comment by Benjamin-eecs** [comment]
With "python setup.py install", same import issues happened as in troubleshooting, ModuleNotFoundError: No module named ‘torchrl._torchrl errors occurs, it means that the C++ extensions were not installed

**Comment by vmoens** [comment]
Are you on Linux or osx?

**Comment by Benjamin-eecs** [comment]
OS: Ubuntu 18.04 LTS (x86_64)
Python version: 3.8.8 (64-bit runtime)
PyTorch version: 1.11.0+cu102

**Comment by Benjamin-eecs** [comment]
> Are you on Linux or osx? Are you using torchrl within the folder? If you install using `install` you should have that very same error if you do
> 
> ```
> cd ~/path/to/torchrl
> python -c “import torchrl”leads to that issue
> ```
> 
> but not with
> 
> ```
> cd ~/somewhere/else
> python -c “import torchrl”leads to that issue
> ```

Yes, the error went away when the directory changed

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ pip install "git+https://github.com/pytorch/functorch.git"
 Go to the directory where you have cloned the torchrl repo and install it
 ```
 cd /path/to/torchrl/
-python setup.py install
+python setup.py develop
```
Comment (vmoens) [inline README.md:85]: Goes away if you import torchrl with a PYTHONPATH that does not coincide with the repo location right?

**Review by vmoens — CHANGES_REQUESTED** [review]
If the error goes away I would suggest to keep the `install` if you don't mind :) 
But I'd be willing to edit the README to emphasise that importing torchrl must be done from somewhere else!

Perhaps we could add that installation in `CONTRIBUTING.md`

File: README.md [hunk 2]
```diff
@@ -111,7 +111,15 @@ pip install pytest
 
 **Troubleshooting**
 
-If a `ModuleNotFoundError: No module named ‘torchrl._torchrl` errors occurs, it means that the C++ extensions were not installed. This may be caused by several dependency issues: cmake, gcc or ninja versioning, or absence of the CuDNN library when working in a CUDA environment. Os MacOs, we recommend installing XCode first.
+If a `ModuleNotFoundError: No module named ‘torchrl._torchrl` errors occurs, it means that the C++ extensions were not installed or not found. 
+One common reason might be that you are trying to import torchrl from within the git repo location. Indeed the following code snippet should return an error if torchrl has not been installed in `develop` mode:
```
Comment (vmoens) [inline README.md:115]: @Benjamin-eecs what do you think?

File: README.md (hunk 1 above)
Comment (Benjamin-eecs) [reply README.md:85]: Yes.

**Comment by Benjamin-eecs** [comment]
> must

For sure.

**Comment by Benjamin-eecs** [comment]
> If the error goes away I would suggest to keep the `install` if you don't mind :) But I'd be willing to edit the README to emphasise that importing torchrl must be done from somewhere else!
> 
> Perhaps we could add that installation in `CONTRIBUTING.md`

For sure.

**Review by vmoens — APPROVED** [review]
LGTM thanks a lot for reporting!

**Comment by bkpcoding** [comment]
Well I get this error even when executed from different location. 
I am using Ubuntu 22.04 LTS (x86_64)
Python version: 3.9.12

**Comment by Benjamin-eecs** [comment]
> Well I get this error even when executed from different location. I am using Ubuntu 22.04 LTS (x86_64) Python version: 3.9.12

Then `python setup.py develop` or `pip install -e .`

**Comment by vmoens** [comment]
> Well I get this error even when executed from different location. I am using Ubuntu 22.04 LTS (x86_64) Python version: 3.9.12

@bkpcoding thanks for reporting
can you print the log of the installation, with the command line you used?

**Comment by bkpcoding** [comment]
image
I get this error when I use 'pip install -e .' or 'python setup.py develop'

**Comment by Benjamin-eecs** [comment]
> image I get this error when I use 'pip install -e .' or 'python setup.py develop'

Yeah, you need cuda and cudnn installed

**Comment by bkpcoding** [comment]
Sure

**Pull request merged**